packages/caliper-samples/benchmark/marbles/config.yaml

```diff
@@ -13,6 +13,11 @@
 #
 
 ---
+flow:
```

Comment:
Shouldn't this be in the `default.yaml` config file? That's where the `Config` class reads it from (as a last resort).
Plus these names don't correspond to the read configs (start-only, etc.). I prefer the skip-x approach since those can be combined in any way.